.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/devcontainers/python:3-bookworm
```

Comment:
Any chance we can use a standard `python` base image here? If not, what does the Microsoft-supplied base image add that enhances use of devcontainers?